File: geometry/circles/python/CircleFormula.py
import math

##Returns True if the point lies on the circle, else false
def isPointOnCircle(center_x, center_y, radius, point_x, point_y):
    return (point_x-center_x)**2+(point_y-center_y)**2-radius**2==0

##Gets radius of circle with the center coords and the coords of a point on the circle
def getRadius(center_x, center_y, point_x, point_y):
    return ((point_x-center_x)**2+(point_y-center_y)**2)**(1/2)

##returns the length of arc given the angle in degrees and radius
def arc_length(radius,angle):
    return (angle*radius*math.pi*(1/180))

File: geometry/circles/javascript/CircleFormula.js
// Returns 'true' if the point lies on the circle, else return 'false'
// ES5
function isPointOnCircle(centerX, centerY, radius, pointX, pointY){
  return Math.pow((pointX - centerX), 2) + Math.pow((pointY - centerY), 2) - Math.pow(radius, 2) == 0
}
// ES6
let isPointOnCircle = (centerX, centerY, radius, pointX, pointY) => Math.pow((pointX - centerX), 2) + Math.pow((pointY - centerY), 2) - Math.pow(radius, 2) == 0

// Gets radius of circle with the center coords and the coords of a point on the circle
// ES5
function getRadius(centerX, centerY, pointX, pointY){
  return Math.pow((Math.pow((pointX - centerX), 2) + Math.pow((pointY - centerY), 2)), (1 / 2))
}
// ES6
let getRadius = (centerX, centerY, pointX, pointY) => Math.pow((Math.pow((pointX - centerX), 2) + Math.pow((pointY - centerY), 2)), (1 / 2))

// Returns the length of arc given the angle in degrees and radius
// ES5
function arcLength(radius, angle){
  return angle * radius * Math.PI * (1 / 180)
}
// ES6
let arcLength = (radius, angle) => angle * radius * Math.PI * (1 / 180)